experimental/packages/otlp-grpc-exporter-base/package.json

```diff
@@ -74,7 +73,8 @@
     "@grpc/grpc-js": "^1.7.1",
     "@grpc/proto-loader": "^0.7.3",
     "@opentelemetry/core": "1.11.0",
-    "@opentelemetry/otlp-exporter-base": "0.37.0"
+    "@opentelemetry/otlp-exporter-base": "0.37.0",
+    "protobufjs": "^7.2.2"
```

Comment:
Adding this as a peer dependency means the grpc exporters also need to take it as a dependency right?